node/src/neighborhood/gossip_acceptor.rs

```diff
@@ -3593,21 +3593,30 @@ mod tests {
     fn standard_gossip_containing_unfamiliar_node_addrs_leads_to_them_being_ignored() {
         /*
 
+        <---- Databases before the gossip ---->
+
         Destination Database (Root) ==>
 
-            Root
-            / \
-           A---B---D
+          Root
+         /  |
+        B---A
+        |
+        D
 
         Source Database (A) ==>
-        - Source has updated version of A and B.
-        - C will not reveal IP, but B, E and F will do.
 
-           Root
+          Root
          /  |
         B---A---E
         |   |
         C   F
+
+        <------------------------------------->
+
+        Source Node will send a gossip to the destination Node with
+        an updated version of Node A and B. The IP of Node C won't
+        be revealed but for Nodes B, E and F, it will be revealed.
```

Comment:
This comment might be too much compared to what it says about the test itself, which is low. Maybe we don't talk about this things because as per my understanding this test is actually about the Node D.

Also, if you insist on the information about B, E, F with address revealed...check it with the GossipBuilder, it was sadly set up differently. 
I caught myself finding out that I don't really know what the flag "reveal address" really means, because Root should definitely not know the addresses of E and F. Right?

Reply:
I think it's better to remove any written text. Since, in the end the dev has to rely on the test.